src/ice_shelf/MOM_ice_shelf_dynamics.F90

```diff
@@ -533,6 +534,7 @@ subroutine initialize_ice_shelf_dyn(param_file, Time, ISS, CS, G, US, diag, new_
     endif
 
     ! initialize basal friction coefficients
+   if (new_sim) then
```

Comment:
Please follow the MOM6 coding convention of using a 2-point indent for logical blocks, as described in the MOM6 style guide at https://github.com/NOAA-GFDL/MOM6/wiki/Code-style-guide.  Doing so greatly improves the readability of the code.

Reply:
The indentation change have been made to this block of code.